mypy.ini

```diff
@@ -0,0 +1,14 @@
+[mypy]
+check_untyped_defs = true
+disallow_incomplete_defs = true
+disallow_untyped_defs = true
```

Comment:
I am ok to try this, and changing them to `false` if future PRs start to not pass because of this.

Is there a way to change this to warnings instead of breaking?
...or the tool can be an "optional" on the list of checks, idk.

Reply:
> I am ok to try this, and changing them to false if future PRs start to not pass because of this.

These particular settings cause mypy to fail when methods/functions are added with missing type hints.

The "correct" course of action would be to add the requested hints, instead of ignoring or disabling. :)

In any case, if you get problems from mypy in the future, feel free to ping me in the PR and I will be happy to assist.